scripts/eurostat/health_determinants/fruits_vegetables/process.py

```diff
@@ -0,0 +1,219 @@
+
+
+class EuroStatFruitsVegetables(EuroStat):
+    """
+    This Class has requried methods to generate Cleaned CSV,
+    MCF and TMCF Files.
+    """
+    _import_name = "fruits_vegetables"
+
+    _mcf_template = ("Node: dcid:{sv}"
+                     "\n{sv_name}"
+                     "\ntypeOf: dcs:StatisticalVariable"
+                     "\npopulationType: dcs:Person"
+                     "{denominator}"
+                     "{education}"
+                     "{residence}"
+                     "{n_portion}"
+                     "{gender}"
+                     "{countryofbirth}"
+                     "{citizenship}"
+                     "{lev_limit}"
+                     "{coicop}"
+                     "{frequenc_fruitsvegetables}"
+                     "{incomequin}"
+                     "\nstatType: dcs:measuredValue"
+                     "\nmeasuredProperty: dcs:count\n")
+
+    _sv_properties_template = {
+        "education":
+            "\neducationalAttainment: dcs:{property_value}",
+        "residence":
+            "\nplaceOfResidenceClassification: dcs:{property_value}",
+        "n_portion":
+            "\nconsumptionQuantity: [{property_value}]",
+        "gender":
+            "\ngender: dcs:{property_value}",
+        "countryofbirth":
+            "\nnativity: dcs:{property_value}",
+        "citizenship":
+            "\ncitizenship: dcs:{property_value}",
+        "lev_limit":
+            "\nglobalActivityLimitationindicator: dcs:{property_value}",
+        "frequenc_fruitsvegetables":
+            "\nactivityFrequency: dcs:{property_value}",
+        "coicop":
+            "\nhealthBehavior: dcs:{property_value}",
+        "incomequin":
+            "\nincome: [{property_value}]",
+        "healthbehavior":
+            "\healthBehaviour: dcs:{property_value}",
+        "healthbehavior_bmi":
+            "__{property_value}",
+    }
+
+    _sv_value_to_property_mapping = {
+        "FruitsVegetables": "healthbehavior",
+        "Male": "gender",
+        "Female": "gender",
+        "Education": "education",
+        "Percentile": "incomequin",
+        "Urban": "residence",
+        "Rural": "residence",
+        "Limitation": "lev_limit",
+        "Daily": "frequenc_fruitsvegetables",
+        "Never": "frequenc_fruitsvegetables",
+        "AtLeastOnceADay": "frequenc_fruitsvegetables",
+        "AtLeastTwiceADay": "frequenc_fruitsvegetables",
+        "NeverOrOccasionallyUsage": "frequenc_fruitsvegetables",
+        "From1To3TimesAWeek": "frequenc_fruitsvegetables",
+        "From4To6TimesAWeek": "frequenc_fruitsvegetables",
+        "OnceADay": "frequenc_fruitsvegetables",
+        "LessThanOnceAWeek": "frequenc_fruitsvegetables",
+        "ForeignBorn": "countryofbirth",
+        "Native": "countryofbirth",
+        "Percentile": "incomequin",
+        "ForeignWithin": "citizenship",
+        "ForeignOutside": "citizenship",
+        "Citizen": "citizenship",
+        "weight": "healthbehavior_bmi",
```

Comment:
`weight` is all lowercase, compared with other keys which start with a capital letter. is that expected?

Reply:
It is as expected.

Reply:
where do these values come from? please add comments for this and the other dictionaries.

Reply:
Added comment.